### ISSUE TITLE
error: multiple input filenames provided

### ISSUE THREAD

**Issue by asomers** [dtolnay/cargo-expand#32]
`cargo expand` with a stable or beta toolchain doesn't work at all, even on a newly created, empty crate.  For example:
```
somers@rustup /tmp> cargo new dummy
     Created binary (application) `dummy` project
somers@rustup /tmp> cd dummy
somers@rustup /t/dummy> cargo expand
    Checking dummy v0.1.0 (file:///tmp/dummy)
error: multiple input filenames provided
error: Could not compile `dummy`.
To learn more, run the command again with --verbose.
```
No variation of `cargo expand --verbose` produces any different output.  Using `cargo +nightly expand` works, but according to the README that shouldn't be necessary.  The README says that `cargo expand` can be invoked with any Rust toolchain, as long as nightly is also installed.

**Comment by dtolnay** [comment]
Thanks! I fixed this in c6960eaf61f6bf0f8b0e82824d52cb7e3ebbb9c1 and released 0.3.17.

**Comment by dtolnay** [comment]
Uh-oh, it no longer builds on stable. Hold on. :laughing: 

```console
error[E0277]: the trait bound `std::ffi::OsString: std::cmp::PartialEq<&str>` is not satisfied
  --> src/main.rs:14:37
   |
14 |     if env::args_os().any(|arg| arg == "--version") {
   |                                     ^^ can't compare `std::ffi::OsString` with `&str`
   |
   = help: the trait `std::cmp::PartialEq<&str>` is not implemented for `std::ffi::OsString`
```

**Comment by dtolnay** [comment]
I believe this is fixed in 0.3.18!
